compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -87,6 +87,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val XfatalWarnings: Setting[Boolean] = BooleanSetting("-Xfatal-warnings", "Fail the compilation if there are any warnings.")
   val XverifySignatures: Setting[Boolean] = BooleanSetting("-Xverify-signatures", "Verify generic signatures in generated bytecode.")
   val XignoreScala2Macros: Setting[Boolean] = BooleanSetting("-Xignore-scala2-macros", "Ignore errors when compiling code that calls Scala2 macros, these will fail at runtime.")
+  val XimportSuggestionTimeout: Setting[Int] = IntSetting("-Ximport-suggestion-timeout", "Timeout (in ms) for searching for import suggestions when errors are reported.", 8000)
```

Comment:
@tgodzik You may want to set this timeout to a very low or even zero value when using the compiler from metals.

Reply:
Thanks for the hint! I think we can set it to 0 now, since we don't use that information in the IDE currently.